pkg/iptables/iptables.go

```diff
@@ -275,6 +275,14 @@ func Exists(table Table, chain string, rule ...string) bool {
 
 // Call 'iptables' system command, passing supplied arguments
 func Raw(args ...string) ([]byte, error) {
+	if firewalldRunning {
+		output, err := Passthrough(Iptables, args...)
+		if err != nil && strings.Contains(err.Error(), "was not provided by any .service files") {
+			// Firewalld is actually not running for some reason
```

Comment:
This empty block is surprising, we might as well invert the condition and `return output, err`.